react/features/base/participants/functions.ts

```diff
@@ -210,7 +210,7 @@ export function getNormalizedDisplayName(name: string) {
  * @returns {(Participant|undefined)}
  */
 export function getParticipantById(
-        stateful: IStore | Function, id: string): Participant|undefined {
+        stateful: IStore | Function| IState, id: string): Participant|undefined {
```

Comment:
Note for the future, maybe we should define a "
state-ish" type and use it across...